docs/notebooks/max_value_entropy_search.pct.py

```diff
@@ -0,0 +1,159 @@
+# %% [markdown]
```

Comment:
A notebook could be a nice way to demonstrate the new acquisition rule, but it could be in a different PR. We're aiming to keep PRs minimal. @vpicheny also suggested we could compare EI and max value entropy search in a single notebook, perhaps the existing intro notebook

Reply:
that's exactly what I had in mind

Reply:
Nice. I will try that in another PR.